.github/workflows/ci.yml

```
@@ -17,8 +17,8 @@ jobs:
 
     strategy:
       matrix:
-        elixir: [1.18.1]
-        otp: [27.0]
+        elixir: [1.19.0-rc.0]
```

```suggestion
        elixir: [1.15, 1.16, 1.17, 1.18, 1.19.0-rc.0]
```
should ensure that library supports older language versions unless there is a strong reason not to.

---

should not target an rc version until it is stable.